### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by wangzeyu135798** [VITA-Group/Adversarial-Contrastive-Learning#2]
Hi:
    In train_simCLR.py, from line 222 to line 224
            inputs_adv = PGD_contrastive(model, inputs, iters=args.pgd_iter, singleImg=False)
            features_adv = model.train()(inputs_adv, 'pgd')
            features = model.train()(inputs, 'normal')
            loss = (nt_xent(features) + nt_xent(features_adv))/2
    In paper figure1, the illustration of workflow comparison. I think the code only express  (d) Dual Stream mode while mode(a) (b) (c) are ignored in code. Right?

**Comment by geekJZY** [comment]
Yes, we only released the code for DS, which is our best solution.
